producer/start-producer.sh

```diff
@@ -0,0 +1,2 @@
+docker build -t my-kafka-producer .
```

Comment:
could be better remove the `build` inside the start script or handle it appropriately

Reply:
Done as described above